CineElorrietaMVC/src/main/java/Controlador/ControladorPanelFacturas.java

```diff
@@ -35,36 +66,173 @@ public void mostrarPanelFacturas() {
 	public void accionadoBottonVolverPanelPrincipal() {
 		this.controlador.navegarPanelPrincipal();
 		ListaProductos listaProd = modelo.getListaTemporal();
-		listaProd.limpiarListTemporal();
+		listaProd.limpiarLista();
 	}
-	
+
 	public String accionadoBotonAnnadirProducto(String producto) {
```

Comment:
este método ( y otros)  lo tenéis en 3 controladores, unificarlo

Reply:
Corregido.

Reply:
Corregido.